README.md

```diff
@@ -77,7 +75,6 @@ package { 'Pester':
   provider => 'powershellcore',
 }
 ```
-
```

Comment:
Is it intentional to leave no whitespace between the code block and the header? Probably renders fine.